workflow/controller/operator.go

```diff
@@ -1151,7 +1151,10 @@ func printPodSpecLog(pod *apiv1.Pod, wfName string) {
 // and returns the new node status if something changed
 func (woc *wfOperationCtx) assessNodeStatus(pod *apiv1.Pod, old *wfv1.NodeStatus) *wfv1.NodeStatus {
 	new := old.DeepCopy()
-	tmpl := woc.GetNodeTemplate(old)
+	tmpl, err := woc.GetNodeTemplate(old)
+	if err != nil {
+		return nil
```

Comment:
should we log here? maybe hard to debug if weird things happen later

Reply:
added an extra log line on this level just in cas